CORE_CONTRIBUTOR.md

```diff
@@ -0,0 +1,35 @@
+# _fastlane_ Core Contributor Expectations
+
+
+We believe that our most valuable resource is the passionate community members who keep _fastlane_ running. We are always looking for active, passionate members of the _fastlane_ community to become core contributors. We hope to harness the diverse experiences of our community to optimize _fastlane_ as the defacto tool to deploy betas and releases for iOS and Android apps.
+
+
+## How does one become a core contributor?
+Contributors who have displayed lasting commitment to the evolution and maintenance of _fastlane_ will be invited to become core contributors. For instance, contributors who:
+- Love to help out other users with issues on GitHub
+- Continue to make _fastlane_ a stable product and encourage features aligned with the _fastlane_ [vision](https://github.com/fastlane/fastlane/blob/master/VISION.md)
+
+
+## As core contributors, we:
+- Review and merge pull requests
+- Respond to issues and help others
+- Own regressions caused by our contributions and PR approvals
+- Maintain consistent coding standards
+- Inform maintainers when critical fixes are merged so a release can be prepared
+- Identify other _fastlane_ community members who would be effective core contributors
+_ Make _fastlane_ the best open source tool and community out there
+
+
+## As core contributors, we always:
+- Ensure that new contributions fit into [_fastlane_’s vision](https://github.com/fastlane/fastlane/blob/master/VISION.md)
```

Comment:
Here we say _fastlane_'s vision, but above we use 'the _fastlane_ vision', I think I like the latter more.